static/app/views/metrics/utils/index.spec.tsx

```diff
@@ -152,6 +152,7 @@ describe('ensureQuotedTextFilters', () => {
     expect(
       ensureQuotedTextFilters('transaction.duration:100', {
         numericKeys: new Set([]),
+        durationKeys: new Set([]),
```

Comment:
Duration keys are treated as numbers as well so overriding it here because `transaction.duration` is a duration key